src/components/CreateToDo/CreateToDo.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import "./CreateToDo";
+import "./CreateToDo.scss";
+import { BiPlusCircle } from "react-icons/bi";
+
+const CreateToDo = ({
+  onAdd,
+  stopAdd,
+}: {
+  onAdd: () => void;
+  stopAdd: () => boolean;
+}) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const getBackGroundColor = () => {
+    if (isHovered) {
+      return isHovered ? "#fbe5d2" : "#f0f1f3";
+    } else {
+      return "white";
+    }
+  };
+
+  const handleMouseEnter = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovered(false);
+  };
+
+  return (
+    <div>
+      {/* {hasNull ? (
```

Comment:
Could you please remove this comment section?

Reply:
Already done.